content/english/image-manipulation/image-filters.md

```diff
@@ -13,24 +13,25 @@ To use the <b>filter()</b> function, you can specify the ImageFilter type within
 
 Add something like this below your previously added code
 
-<pre>
-    blurredImage = image.filter(ImageFilter.BLUR) 
-    blurredImage.save("myCatBlurred.jpg")
-</pre>
+```python
+blurredImage = image.filter(ImageFilter.BLUR) 
+blurredImage.save("myCatBlurred.jpg")
+```
 
 You should see the saved cat image is blurred. Try this with a few other image filters until you find one you like!
 
-My blurred and rotated image now looks like this:
-![alt text](../media/blurred_upside_down.png "blurred cat upside down")
+I want to make my image a little extra blurry, so I'm going to use `ImageFilter.BoxBlur(15)` as my filter instead of `ImageFilter.BLUR`. My blurred and rotated image now looks like this:
+<!-- ![alt text](../media/blurred_upside_down.png "blurred cat upside down") -->
```

Comment:
Not sure what the change is here that is separate from my own, but going through and making sure that I'm using `` instead :) 

I'm using the <img> tag because can scale the image down to 50% instead of max size.

Reply:
Just kidding I see the comment is to delete the comment, done! thanks